### PR TITLE
Update ITEM_en.tsv

### PR DESCRIPTION
**Pull request by emailboxu**
~3700

### CODE REVIEW

File: ITEM_en.tsv [hunk 1]
```diff
@@ -3645,59 +3645,59 @@ ITEM_20150317_003876	A dangerous bomb that will explode soon after being thrown.
...
-ITEM_20150323_003940	This is the material to make a Safi Necklace. You can earn from Rikius of Sirdgele Forest.
-ITEM_20150323_003941	This is the material to make a Safi Necklace. You can earn from Bramble of Kvailas Forest.
-ITEM_20150323_003942	You can earn from Yonajolem of Ramsties canyon.
-ITEM_20150323_003943	You can earn from Scorpio of Akmens Ridge.
-ITEM_20150323_003944	You can earn from unicorn of Escanicu Village.
-ITEM_20150323_003945	You can earn from Liverfeed of Escanicu Village.
-ITEM_20150323_003946	You can earn from Devilglove of Fedimian's outside.
-ITEM_20150323_003947	You can earn from Moa of Fedimian's outside.
-ITEM_20150323_003948	It is an empty flask. It is able to save the liquid. You can buy it from the Alchemist Master.
+ITEM_20150323_003938	Can be obtained from the Colimencia quest in Kobalt Forest.
```
Comment (Sunaries) [inline ITEM_en.tsv]: _Can be obtained from Colimencia of the Kobalt Forest quest._

This means that there is a quest in Kobalt Forest involving the boss monster Colemencia where you can acquire the item from.

File: ITEM_en.tsv [hunk 2]
```diff
@@ -3645,59 +3645,59 @@ ITEM_20150317_003876	A dangerous bomb that will explode soon after being thrown.
...
+ITEM_20150323_003939	Can be obtained from the Merge quest in Septyni Glen.
```
Comment (Sunaries) [inline ITEM_en.tsv]: _Can be obtained from Merge of the Septyni Valley quest_

This means that there is a quest in Septyni Valley involving the boss monster Merge where you can acquire the item from.
Although it says 셉티니 골 is Septyni Glen in the wiki, I believe the correct name of this map is Septyni Valley.

File: ITEM_en.tsv [hunk 3]
```diff
@@ -3645,59 +3645,59 @@ ITEM_20150317_003876	A dangerous bomb that will explode soon after being thrown.
...
+ITEM_20150323_003940	An ingredient to make the Safi Necklace. Can be obtained from Rikaus in Sirdgela Forest.
```
Comment (Sunaries) [inline ITEM_en.tsv]: Safi -> Saffie

File: ITEM_en.tsv [hunk 4]
```diff
@@ -3645,59 +3645,59 @@ ITEM_20150317_003876	A dangerous bomb that will explode soon after being thrown.
...
+ITEM_20150323_003941	An ingredient to make the Safi Necklace. Can be obtained from Bramble in Kvailas Forest.
```
Comment (Sunaries) [inline ITEM_en.tsv]: Safi -> Saffie

File: ITEM_en.tsv [hunk 5]
```diff
@@ -3645,59 +3645,59 @@ ITEM_20150317_003876	A dangerous bomb that will explode soon after being thrown.
...
-ITEM_20150323_003931	Sandwich meat
+ITEM_20150317_003906	The Scripture that anyone calling themselves a Pilgrim of the Cathedral would carry with them.
+ITEM_20150317_003907	Test Soul Reagent
+ITEM_20150317_003908	Reagent for soul testing that Graschius gave you. The key is to secretly spray it without drawing the monster's attention.
+ITEM_20150317_003909	Cold Monster Fluid
+ITEM_20150317_003914	A piece of paper swallowed by a monster. It looks like someone's portrait.
+ITEM_20150317_003916	Soldiers' Relics
+ITEM_20150317_003917	The relics of soldiers who died protecting the Miners' Village.
+ITEM_20150317_003918	Miners' Village Food
+ITEM_20150317_003919	Food that Vubbes plundered in the Miners' Village. Some of it has been eaten.
```
Comment (Sunaries) [inline ITEM_en.tsv]: Food that **the** Vubbes plundered in the Miners' Village. Some of it has been eaten.

File: ITEM_en.tsv [hunk 6]
```diff
@@ -3645,59 +3645,59 @@ ITEM_20150317_003876	A dangerous bomb that will explode soon after being thrown.
...
-ITEM_20150317_003885	Research of Priest
-ITEM_20150317_003886	Research collected by Priest
-ITEM_20150317_003887	Hind Leg of Dumaro
-ITEM_20150317_003888	A part has a lot of meat among Dumaro's hind leg
-ITEM_20150317_003889	Leather front plate
+ITEM_20150317_003884	A lengthy advertisement noting the Kingdom's honor and pride
+ITEM_20150317_003885	Priest's Research
+ITEM_20150317_003886	Research collected by the Priest
+ITEM_20150317_003887	Dumaro's Hind Leg
+ITEM_20150317_003888	The part of a dumaro that has the most meat
```
Comment (Sunaries) [inline ITEM_en.tsv]: dumaro -> Dumaro

Assuming this is a type of mob.

**Comment by imcgames** [comment]
Thank you at all times :+1:

**Pull request merged**